spec/std/big/big_int_spec.cr

```diff
@@ -296,13 +296,13 @@ describe "BigInt" do
   it "can be casted into other Number types" do
     big = BigInt.new(1234567890)
     big.to_i.should eq(1234567890)
-    big.to_i8.should eq(-46)
-    big.to_i16.should eq(722)
+    big.to_i8!.should eq(-46)
```

Comment:
Is this intended?

Reply:
I think so, the same happens when you do `1234567890.to_i8`, it just truncates the contents to the right-most 8 bits (`11010010`) and reinterprets that as `Int8`, and it ends up negative because of the left-most `1`.

Reply:
It is intended as in "to keep the current behavior"

Reply:
Sry, that was too unspecific. I meant the change. It removes test coverage of `big.to_i8` etc.

Reply:
It's because it will overflow on future changes. The coverage for the semantics of the numbers are not that great, and am not focusing that much in Big